dev/devicelab/bin/tasks/build_aar_module_test.dart

```diff
@@ -88,7 +88,7 @@ Future<void> main() async {
       section('Build release AAR');
 
       await inDirectory(projectDir, () async {
-        await flutter('build', options: <String>['aar', '--verbose']);
```

Comment:
You might consider keeping this one. If an aar fails to build then I think verbose logs might be required to know why.

Reply:
I suspect if it's important we can add it back 🤷🏼